uw-frame-components/css/angular.less

```diff
@@ -1,6 +1,6 @@
-@import "../bower_components/normalize-less/normalize.less";
-@import "../../bower_components/font-awesome/css/font-awesome.css";
-@import "../bower_components/bootstrap/less/bootstrap.less";
+@import "https://raw.githubusercontent.com/additiveinverse/normalize.less/master/normalize.less";
```

Comment:
I would not want to change dependencies in this same revision, but Bootstrap includes a normalize.less - is that just as good?